public/locales/fr/translation.json

```
@@ -0,0 +1,79 @@
+    "multi_seed": "Multi semence",
+    "wrong_seed_not_in_list": "Mauvaise semence non dans la liste",
+    "error_deleting_directory": "Erreur lors de la suppression du répertoire, voir la console pour plus de détails",
+    "error_creating_directory": "Erreur lors de la création du répertoire, voir la console pour plus de détails",
+    "error_fetching_uuid": "Erreur lors de la récupération de l'UUID, voir la console pour plus de détails",
+    "error_fetching_inference_data": "Erreur lors de la récupération des données d'inférence, voir la console pour plus de détails",
+    "please_select_directory": "Veuillez sélectionner un répertoire",
+    "error_reading_azure_storage_directory": "Erreur lors de la lecture du répertoire de stockage Azure, voir la console pour plus de détails",
+    "error_fetching_model_metadata": "Erreur lors de la récupération des métadonnées du modèle, voir la console pour plus de détails",
+    "creative_commons_introduction": "En important vos images sur l'interface de classification des semences, vous acceptez de licencier votre travail sous une licence Creative Commons Attribution-Partage dans les Mêmes Conditions (CC BY-SA). Cet accord décrit les termes et conditions de la licence ainsi que d'autres considérations.",
+    "creative_commons_terms_and_conditions": "Attribution : Vous autorisez les autres à copier, distribuer, afficher et exécuter votre œuvre protégée par le droit d'auteur — et les œuvres dérivées qui en sont basées — mais seulement s'ils vous donnent le crédit approprié en citant votre nom et la source.\nPartage dans les mêmes conditions : Vous permettez aux autres de distribuer les œuvres dérivées uniquement sous une licence identique à celle qui régit votre travail.\nApprentissage automatique : Vous accordez à l'ACIA le droit d'utiliser vos images pour entraîner des modèles d'apprentissage automatique. Ces modèles peuvent être utilisés à diverses fins, y compris la recherche, l'analyse et les activités commerciales.\nGarantie : Vous déclarez et garantissez que vous êtes le propriétaire légal du contenu que vous importez et qu'il ne porte pas atteinte aux droits d'auteur, marques déposées ou autres droits de tiers.\nConsentement : Si votre image comprend des individus identifiables, vous affirmez avoir obtenu leur consentement pour que l'image soit partagée et utilisée selon ces termes.\nRenonciation : L'image est fournie \"telle quelle\". Vous renoncez à toutes garanties, y compris toute garantie concernant l'exactitude de l'image ou son adéquation à un usage particulier.",
+    "creative_commons_acknowledgment": "En cliquant sur \"J'accepte\", vous confirmez que vous avez lu et compris cet accord, et vous serez légalement lié par ses termes et conditions.",
+    "creative_commons_title": "Utilisation des images Creative Commons",
+    "creative_commons_introduction_header": "Introduction",
+    "creative_commons_terms_and_conditions_header": "Termes et Conditions",
```

```suggestion
    "creative_commons_terms_and_conditions_header": "Conditions d'utilisation",
```
https://vitrinelinguistique.oqlf.gouv.qc.ca/fiche-gdt/fiche/26527155/conditions-generales-dutilisation